src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -50,6 +52,7 @@ public class HttpClientRequestMessage : DataServiceClientRequestMessage, ISendin
         private readonly HttpRequestMessage _requestMessage;
         private readonly HttpClient _client;
         private readonly HttpClientHandler _handler;
+        private readonly IHttpClientHandlerProvider _clientHandlerProvider;

```

Comment:
```suggestion
```

I don't think we need a member for this, it seems we only use it in the constructor